src/components/otherAnimalTypes.jsx

```diff
@@ -12,6 +12,7 @@ const OtherAnimalTypes = ({
   OtherAnimalTypes,
   shouldShow,
 }) => {
+  const {otherAnimalTypes}  =rest.formik.values;
```

Comment:
do you have prettier installed? it doesn't look like any of these files are being formatted?

Reply:
yes i do, may be i forgot to save it